Makefile

```diff
@@ -193,7 +193,7 @@ clean: isc-clean
 	rm -f $(FGEN)
 	rm -f sfilet-* snodet-* fidout-*
 	rm -f upFile dwFile
-	rm -f *.o *.d *.d.* help.h
+	rm -f *.o *.d *.d.*
```

Comment:
Just for your information: the licencing header at the top of this file is outdated.

Reply:
Thanks, fixed.